core-container/entry.sh

```diff
@@ -178,6 +239,7 @@ function run_kcp() {
 function run_init() {
     echo "--< Starting init >--"
     wait_kcp_ready
+    set_provider_adapters
```

Comment:
but kflex is not related to kcp why wait for both ?

Reply:
Not sure I follow.  Lets discuss tomorrow morning.  Or open an issue.